README.md

```diff
@@ -33,6 +33,21 @@ On NixOS and nix-darwin, simply replace your `nix.package`, like so:
 ```nix
 nix.package = pkgs.nix-monitored;
 ```
+And then import and enable it like this:
+```nix
+{
+  imports = [
+    inputs.nix-monitored.nixosModules.default
+  ];
+
+  nix.monitored.enable = true;
+}
+```
+
+You can also import it in the overlay below by putting this line inside alongside the rest:
+```
+nix-monitored = inputs.nix-monitored.packages.${self.system}.default.override self;
+```
```

Comment:
This should probably rather be added to the module: https://github.com/ners/nix-monitored/blob/master/flake.nix#L109